src/Helpers/TeamspeakHelper.php

```diff
@@ -21,6 +21,81 @@ class TeamspeakHelper
     public static function connect($tsUsername, $tsPassword, $tsHostname, $tsServerQuery, $tsServerPort)
     {
         return \TeamSpeak3::factory("serverquery://" . $tsUsername . ':' . $tsPassword . '@' . $tsHostname .
```

Comment:
maybe use `sprintf` in order to build the uri.
Also, add use statement for `Teamspeak3`